copier/main.py

```diff
@@ -547,6 +559,7 @@ def run_copy(self) -> None:
 
         See [generating a project][generating-a-project].
         """
+        os.environ["COPIER_TEMPLATE_PATH"] = str(self.template.local_abspath)
```

Comment:
And this? Were you unable to obtain it from elsewhere? Maybe the file system loader, or `_copier_conf.src_path`?

Reply:
Oops, forgot to remove that! Yes I was able to get that from the loader!